skglm/estimators.py

```diff
@@ -1333,5 +1488,5 @@ def path(self, X, Y, alphas, coef_init=None, **params):
         datafit = compiled_clone(self.datafit, to_float32=X.dtype == np.float32)
         penalty = compiled_clone(self.penalty)
 
-        return bcd_solver_path(X, Y, datafit, penalty, alphas=alphas,
-                               coef_init=coef_init, **params)
+        return multitask_bcd_solver_path(X, Y, datafit, penalty, alphas=alphas,
```

Comment:
Is there a reason why we don't call ``_glm_fit`` here?

Reply:
Here, we want to compute the path for several alphas. Hence, calling `multitask_bcd_solver_path` as it is done in the Lasso case with `cd_solver_path`. So I'm guessing it is all right

Reply:
What about the fit method, why don't call ``_glm_fit``? In fact, we handle the case of MLT (c.f. L135).

Reply:
You're right but we don't handle multiple penalty values in the `_glm_fit`.